### ISSUE TITLE
Improve color contrast between non-text elements and background

### ISSUE THREAD

**Issue by cstarcher** [TexasDigitalLibrary/Vireo#1664]
Relevant Components:

- [ ] Document Information
- [ ] File Upload
- [ ] Settings

Description
There is not a significant color contrast between meaningful graphics and the background on which they appear. The visual presentation of active controls/meaningful graphics should have a contrast ratio of at least 3:1 against adjacent color(s): [Success Criterion 1.4.1 Use of Color|https://www.w3.org/TR/WCAG21/#use-of-color].

Current Code
```
<button class="invisible-button">

<!-- ngIf: !fileValidationError-->

<span class="glyphicon glyphicon-upload ng-pristine ng-untouched ng-valid ng-scope ng-empty" ng-if="!fileValidationError" ngf-pattern=".pdf" accept=".pdf" ng-model="fileModel" ngf-multiple="true"></span>

<!-- end ngIf: !fileValidationError-->

<!-- ngIf: fileValidationError-->

</button>
```

Solution
Alter color of glyphicon by adding a darker color css property directly to the style, as in:

```<span class="glyphicon glyphicon-user" style="color:blue"></span>```

OR add it as a class to your icon and set the font color to it in CSS

**Comment by jcreel** [comment]
Relates to #1663
